spec/calculator.spec.js

```diff
@@ -1,24 +1,31 @@
-'use strict';
+(function (factory) {
```

Comment:
Now that this file will be used by the browser as well as Node JS (not just Node JS), we need to use the IIFE that we used before to put our tests inside a "factory function". This is because our tests depend on 2 things. They depend on the Calculator (because they test it), and they also depend on Chai (for the assertions).